### DISCUSSION TITLE
Exporting OrangeFS volumes via SAMBA

### DISCUSSION THREAD

**Discussion by anon314159** [waltligon/orangefs#104] [Q&A] [answered]
While I am aware of the native client for Windows are there any caveats with leveraging a client mount point and then re-exporting via SAMBA/CTDB?  Does OrangeFS support posix acls? Are there any required VFS plugins for better integration with SMB?

**Comment by waltligon** [comment, marked as answer]
Yes, reexports should work - though it’s been a while since we tested that.  Also we support ACLs.

Walt
